[2025-05-22T21:24:59.065+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scrape_and_insert_bank_reviews.copy_tables_to_remote manual__2025-05-19T10:58:39.511715+00:00 [queued]>
[2025-05-22T21:24:59.088+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scrape_and_insert_bank_reviews.copy_tables_to_remote manual__2025-05-19T10:58:39.511715+00:00 [queued]>
[2025-05-22T21:24:59.091+0100] {taskinstance.py:2170} INFO - Starting attempt 4 of 4
[2025-05-22T21:24:59.125+0100] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): copy_tables_to_remote> on 2025-05-19 10:58:39.511715+00:00
[2025-05-22T21:24:59.136+0100] {standard_task_runner.py:60} INFO - Started process 21837 to run task
[2025-05-22T21:24:59.141+0100] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'scrape_and_insert_bank_reviews', 'copy_tables_to_remote', 'manual__2025-05-19T10:58:39.511715+00:00', '--job-id', '1954', '--raw', '--subdir', 'DAGS_FOLDER/S_G_M_R.py', '--cfg-path', '/tmp/tmprypa5m_x']
[2025-05-22T21:24:59.143+0100] {standard_task_runner.py:88} INFO - Job 1954: Subtask copy_tables_to_remote
[2025-05-22T21:24:59.257+0100] {task_command.py:423} INFO - Running <TaskInstance: scrape_and_insert_bank_reviews.copy_tables_to_remote manual__2025-05-19T10:58:39.511715+00:00 [running]> on host DESKTOP-RRUTCI2.
[2025-05-22T21:24:59.756+0100] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='scrape_and_insert_bank_reviews' AIRFLOW_CTX_TASK_ID='copy_tables_to_remote' AIRFLOW_CTX_EXECUTION_DATE='2025-05-19T10:58:39.511715+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-19T10:58:39.511715+00:00'
[2025-05-22T21:24:59.762+0100] {logging_mixin.py:188} INFO - üîÑ Reading 'dim_bank' from local schema 'public_decision_schema'...
[2025-05-22T21:24:59.969+0100] {logging_mixin.py:188} INFO - ‚¨ÜÔ∏è Uploading 'dim_bank' to remote (replace if exists)...
[2025-05-22T21:25:02.110+0100] {logging_mixin.py:188} INFO - ‚úÖ 'dim_bank' copied and replaced successfully.
[2025-05-22T21:25:02.113+0100] {logging_mixin.py:188} INFO - üîÑ Reading 'dim_branch' from local schema 'public_decision_schema'...
[2025-05-22T21:25:02.149+0100] {logging_mixin.py:188} INFO - ‚¨ÜÔ∏è Uploading 'dim_branch' to remote (replace if exists)...
[2025-05-22T21:25:03.102+0100] {logging_mixin.py:188} INFO - ‚úÖ 'dim_branch' copied and replaced successfully.
[2025-05-22T21:25:03.104+0100] {logging_mixin.py:188} INFO - üîÑ Reading 'dim_location' from local schema 'public_decision_schema'...
[2025-05-22T21:25:03.128+0100] {logging_mixin.py:188} INFO - ‚¨ÜÔ∏è Uploading 'dim_location' to remote (replace if exists)...
[2025-05-22T21:25:04.060+0100] {logging_mixin.py:188} INFO - ‚úÖ 'dim_location' copied and replaced successfully.
[2025-05-22T21:25:04.064+0100] {logging_mixin.py:188} INFO - üîÑ Reading 'dim_sentiment' from local schema 'public_decision_schema'...
[2025-05-22T21:25:04.094+0100] {logging_mixin.py:188} INFO - ‚¨ÜÔ∏è Uploading 'dim_sentiment' to remote (replace if exists)...
[2025-05-22T21:25:04.979+0100] {logging_mixin.py:188} INFO - ‚úÖ 'dim_sentiment' copied and replaced successfully.
[2025-05-22T21:25:04.982+0100] {logging_mixin.py:188} INFO - üîÑ Reading 'fact_reviews' from local schema 'public_decision_schema'...
[2025-05-22T21:25:05.029+0100] {logging_mixin.py:188} INFO - ‚¨ÜÔ∏è Uploading 'fact_reviews' to remote (replace if exists)...
[2025-05-22T21:25:06.005+0100] {logging_mixin.py:188} INFO - ‚úÖ 'fact_reviews' copied and replaced successfully.
[2025-05-22T21:25:06.007+0100] {logging_mixin.py:188} INFO - üîÑ Reading 'clean_bank_reviews' from local schema 'public'...
[2025-05-22T21:25:06.015+0100] {logging_mixin.py:188} INFO - ‚ùå Error copying 'clean_bank_reviews': Could not reflect: requested table(s) not available in Engine(postgresql+psycopg2://airflow_user:***@localhost:5432/airflow_db) schema 'public': (clean_bank_reviews)
[2025-05-22T21:25:06.017+0100] {python.py:201} INFO - Done. Returned value was: None
[2025-05-22T21:25:06.048+0100] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=scrape_and_insert_bank_reviews, task_id=copy_tables_to_remote, execution_date=20250519T105839, start_date=20250522T202459, end_date=20250522T202506
[2025-05-22T21:25:06.108+0100] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-22T21:25:06.144+0100] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
